omero_search_engine/validation/results_validator.py

```diff
@@ -747,14 +1029,26 @@ def get_no_images_sql_containers():
             messages.append(
                 "\n-----------------------------------------------------------------------------\n"  # noqa
             )
-    base_folder = "/etc/searchengine/"
-    if not os.path.isdir(base_folder):
-        base_folder = os.path.expanduser("~")
+    if write_report:
+        base_folder = "/etc/searchengine/"
```

Comment:
The location of the base folder is hard coded in a few places.
I think we need to use a configuration variable